frontends/web/src/components/bluetooth/bluetooth.tsx

```diff
@@ -54,10 +55,36 @@ const BluetoothInner = ({ peripheralContainerClassName }: Props) => {
   if (!state) {
     return null;
   }
+
+  if (state.bluetoothUnauthorized) {
+    return (
+      <Message type="warning">
+        <div className={styles.bluetoothDisabledContainer}>
+          <span className={styles.bluetoothDisabledTitle}>
+            {t('bluetooth.disabledPermissionTitle')}
+          </span>
+          <span >
+            {t('bluetooth.disabledPermissionDescription')}
+          </span>
+          <A className={styles.link} href="app-settings:">
+            {t('generic.enable')}
+          </A>
+        </div>
+      </Message>
+    );
+  }
+
   if (!state.bluetoothAvailable) {
```

Comment:
Do we wanna both messages if BLE is turned off and it is not authorized? 

Seems like it could just show the one to turn it on first, and if turned on and unauthorized, show the "Please allow..." message then?